lib/src/repositories/ton_wallet_repository/ton_wallet_repository_impl.dart

```diff
@@ -409,7 +409,7 @@ mixin TonWalletRepositoryImpl implements TonWalletRepository {
       createPoller(
         TonWalletGqlBlockPoller(tonWallet: tonWallet, transport: transport),
       );
-    } else if (transport is ProtoTransport) {
+    } else if (transport is ProtoTransport || transport is JrpcTransport) {
```

Comment:
Add same line in `GenericContarctRepositoryImpl`, there is the same send logic

Reply:
🤟